src/test/java/posters/tests/smoke/BrowseRandomVisualAssertTest.java

```diff
@@ -29,7 +24,7 @@
 @Severity(SeverityLevel.NORMAL)
 @Tag("smoke")
 @Browser("Firefox_headless")
-@Ignore
+// @Ignore
```

Comment:
Please reactivate the @Ignore. This is fine as it was. By the way it also demonstrates ignored test cases in the allure report.